com.vrcfury.vrcfury/Editor/VF/Feature/EditorOnlyBuilder.cs

```diff
@@ -0,0 +1,42 @@
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+using VF.Builder;
+using VF.Feature.Base;
+using VF.Injector;
+using VF.Inspector;
+using VF.Model.Feature;
+
+namespace VF.Feature {
+    [VFService]
+    public class EditorOnlyBuilder : FeatureBuilder {
+        
+        [FeatureBuilderAction(FeatureOrder.RemoveEditorOnly)]
+        public void Apply() {
+            RemoveChildrenWithEditorOnly(avatarObject);
+        }
+
+        private void RemoveChildrenWithEditorOnly(VFGameObject gameObject) {
+            foreach (var child in gameObject.Children()) {
+                if (child.gameObject.CompareTag("EditorOnly")) {
+                    child.Destroy();
+                } else if (child.childCount > 0) {
+                    RemoveChildrenWithEditorOnly(child);
+                }
+            }
+        }
+        
+        public override string GetEditorTitle() {
```

Comment:
This shouldn't have a GetEditorTitle or CreatorEditor, since there is no model associated with this builder, so it's not possible to add as a component anyways.

Reply:
done!